learnr_bank/learnr_set1.Rmd

```diff
@@ -0,0 +1,225 @@
+---
+title: "Question Bank Set 1"
+author: "Shemaiah Sotrah"
+date: "6/24/2020"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(tidyverse)
+library(learnr)
+```
+
+# Resources for Shemaiah:
+
++ **Before you start**, go through this tutorial carefully: [learnr exercises](https://rstudio.github.io/learnr/exercises.html). 
+
++ For these questions, you do not need a setup chunk (but we will once we start using packages!)
++ Each question should have a corresponding `solution` chunk. IN that chunk, do not just type the answer - write out the code to get the answer (e.g. for q1 don't put `30`, put `5*6`)
++ Each question should have a corresponding hint. In this case, all hints are not actually code, but markdown text!! Therefore, you actually need to use HTML to write the hints instead of writing code chunks for hints (described in link above). 
+  + If a hint is ever code, use a chunk instead.
++ I've done set 1 question 1 below to show you exactly what I'm looking for.
+
+
+### Set 1: Using R as a calculator
+
+1. What is 5 times 6?
+
+```{r calc_q1, exercise =TRUE}
+
+```
+  
+<div id="calc_q1-hint">
+**Hint:**  Use the asterisk `*` to multiply numbers
+</div>
+
+2. What is 600 divided by 2.53?
+ 
+``` {r calc_q2, exercise = TRUE}
+
+```
+
+<div id="calc_q2-hint">
+**Hint:**  Use the forward slash `/` to divide numbers
+</div>
+
+3. What is 138 squared?
+  
+``` {r calc_q3, exercise = TRUE}
+
+```
+
+<div id="calc_q3-hint">
+**Hint:**  Use the carot symbol `^` *or* two asterisks (`**`) to raise numbers to an exponent.
+</div>
+
+4. What is the square root of 14?
+  
+``` {r calc_q4, exercise = TRUE}
+
+```  
+
+<div id="calc_q4-hint">
+**Hint:**   Square roots are like raising to the power of 0.5
+</div>
+
+### Answer Key for Set 1
+``` {r, calc-q1-answer}
+#Solution to Question 1
+5*6
+```
+
+``` {r, calc-q2-answer}
+#Solution to Question 2
+600/2.53
+```
+
+``` {r, calc-q3-answer}
+#Solution to Question 3
+138**2 
+138^2 #equally valid approach
+```
+
+``` {r, calc-q4-answer}
+#Solution to Question 4
```

Comment:
Copied wrong solution here!

Reply:
Thank you! I fixed it. I am currently almost done with section 2.